webapp/src/containers/createRequest/index.tsx

```diff
@@ -137,58 +147,71 @@ const CreateRequest = () => {
-                                        target='_blank'
-                                    >
-                                        {Constants.RequestButtonText}
-                                    </a>
-                                    <div className='servicenow-request-button__redirect-text'>
-                                        {Constants.RequestButtonRedirectText}
-                                    </div>
-                                </div>
+                ) : (
+                    <>
+                        <div
+                            className={`padding-h-12 padding-top-25 padding-bottom-30
+                            ${suggestionChosen ? 'height-auto' : 'height-120'}`}
+                        >
+                            <AutoSuggest
+                                placeholder='Search catalog items'
+                                inputValue={autoSuggestValue}
+                                onInputValueChange={handleInputChange}
+                                onChangeSelectedSuggestion={handleRequestSelection}
+                                suggestionConfig={{
+                                    suggestions,
+                                    renderValue: (suggestion) => suggestion.name,
+                                }}
+                                charThresholdToShowSuggestions={Constants.CharThresholdToSuggestRequest}
+                            />
+                            {suggestionChosen && request && (
+                                <>
+                                    <ul className='search-panel__description margin-top-25 padding-left-15 font-14'>
+                                        {Constants.RequestDataLabelConfig.map((header) => (
+                                            <li
+                                                key={header.key}
+                                                className='d-flex align-items-center search-panel__description-item margin-bottom-10'
+                                            >
+                                                <span className='search-panel__description-header margin-right-10 text-ellipsis'>{header.label}</span>
+                                                <span className='search-panel__description-text channel-text wt-500 text-ellipsis white-space-inherit'>{showModalLoader ? <SkeletonLoader/> : request[header.key] || 'N/A'}</span>
+                                            </li>
+                                        ))}
+                                    </ul>
+                                    {serviceNowBaseURL && (
+                                        <div>
+                                            <a
+                                                className='color--link btn btn-primary servicenow-request-button'
+                                                href={`${serviceNowBaseURL}/${Constants.REQUEST_BASE_URL}${request.id}`}
```

Comment:
```suggestion
                                                href={`${serviceNowBaseURL}/${Constants.REQUEST_BASE_URL}/${request.id}`}
```
Remove `/` from the end of the constant `REQUEST_BASE_URL`.

Reply:
there is no `/` at end of URL, we are using id as a param

Reply:
there is no `/` at end of URL, we are using id as a param